Samples/Caboodle.Samples/ViewModel/PhoneDialerViewModel.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Windows.Input;
+using Microsoft.Caboodle;
+using Xamarin.Forms;
+
+namespace Caboodle.Samples.ViewModel
+{
+    public class PhoneDialerViewModel : BaseViewModel
+    {
+        string phoneNumber;
+
+        public PhoneDialerViewModel()
+        {
+            OpenPhoneDialerCommand = new Command(OnOpenPhoneDialer);
+        }
+
+        public ICommand OpenPhoneDialerCommand { get; }
+
+        public string PhoneNumber
+        {
+            get => phoneNumber;
+            set => SetProperty(ref phoneNumber, value);
+        }
+
+        async void OnOpenPhoneDialer()
+        {
+            try
+            {
+                PhoneDialer.Open(PhoneNumber);
+            }
+            catch (Exception)
+            {
+                await Application.Current.MainPage.DisplayAlert("Error", "Dialer is not supported.", "OK");
```

Comment:
I added a method `BaseViewModel.DisplayAlert` so you shouldn't need to get the current page.